doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -392,7 +441,7 @@ To configure the Hash algorithms, set one or more of the following Kconfig optio
 +-----------------------+---------------------------------------------------+
 | Hash algorithm        | Configuration option                              |
 +=======================+===================================================+
-| SHA-1                 | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_1`       |
+| SHA-1 (unsafe)        | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_1`       |
```

Comment:
```suggestion
| SHA-1 (weak)        | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_1`       |
```

same elsewhere. See Georgios's review.

Reply:
Agree the PSA Crypto API doc also uses the word weak for it I think it would be nice to be aligned with them, and we use weak in L460/461
https://armmbed.github.io/mbed-crypto/html/api/ops/hashes.html